src/features/courses/course-controller.ts

```diff
@@ -72,11 +74,26 @@ class CourseController {
         });
     }
 
-    getTopicById(id: number): Promise<CourseTopicContent> {
+    getTopicById(id: number, userId: number | undefined = undefined): Promise<CourseTopicContent> {
+        const include = [];
+        if (userId !== undefined) {
```

Comment:
Nit: I prefer _.isNil, even though right now it can't be null if we ever made so the case in the future it would "just work"
If null ever did get in here it would be a problem right?
Or maybe we do want it this way because there is a reason that null should get through if we allow it in the future?

Reply:
I think that, in terms of getting a topic with a userId associated that null probably means the same as undefined, so I'll make that change.